xwiki-platform-core/xwiki-platform-attachment/xwiki-platform-attachment-api/src/test/java/org/xwiki/attachment/internal/listener/MovedAttachmentListenerTest.java

```diff
@@ -95,16 +95,15 @@ void onEventNoUpdateReference()
     }
 
     @Test
-    void onEvent()
+    void onEvent() throws RefactoringException
```

Comment:
It may be just me but I'm always throwing `Exception` and never anything more specific. This makes refactoring easier.

Reply:
Not sure I'm a big fan of that logic, if you push it you can end up having to put `throws RefactoringException` on every single test. I also like making sure that it's excepted to have those exceptions.